src/js/TodoList.js

```diff
@@ -0,0 +1,246 @@
+/* eslint-disable class-methods-use-this */
+import { BASE_URL, USER_PATH, USERLIST, $ } from './constants.js';
+
+class TodoList {
+  constructor() {
+    this.userInfos = {};
+    this.init();
+  }
+
+  createNonPriorityElement(userTodoListElement) {
+    return `
+			<select class="chip select">
+			<option value="NONE selected">순위</option>
+			<option value="FIRST">1순위</option>
+			<option value="SECOND">2순위</option>
+			</select>
+			${userTodoListElement.contents}
+		`;
+  }
+
+  createPriorityElement(userTodoListElement) {
+    return `
+			<span class="
+				chip ${userTodoListElement.priority === 'FIRST' ? 'primary' : 'secondary'}">
+				${userTodoListElement.priority === 'FIRST' ? '1순위' : '2순위'}
+			</span>
+			${userTodoListElement.contents}
+		`;
+  }
```

Comment:
이 부분은 ``Object Literal``로 관리하시는건 어떠실까요?
```suggestion
  const priorityElement = {
    NONE: (content) => `<select class="chip select">
			<option value="NONE selected">순위</option>
			<option value="FIRST">1순위</option>
			<option value="SECOND">2순위</option>
			</select>
			${cotent}`;
    FIRST: (content) => `<span class="chip primary">
                        1순위
                        </span>
			${content}`;
	SECOND: (content) => `<span class="chip secondary">
	                 2순위
                         </span>
                       ${content}`;
  }
  
  
  // 아래의 삼항연산자 대체
  priorityElement[userTodoListElement.priority](userTodoListElement.content);
```

이렇게 작성할 수 있겠네요!

Reply:
와, 좋은 방법이네요!! 이런 걸 `object literal`이라 하는군요! 많이 배워갑니다 :-) 감사합니다